BUILD.bazel

```diff
@@ -3932,15 +3939,18 @@ ALL_NEONDOT_MICROKERNEL_SRCS = [
     "src/qu8-gemm/gen/8x8c4-minmax-rndnu-neondot.c",
     "src/qu8-gemm/gen/8x16c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/1x8c4-minmax-rndnu-neondot.c",
+    "src/qu8-igemm/gen/1x16c4-minmax-fp32-neondot.c",
     "src/qu8-igemm/gen/1x16c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/1x32c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/2x8c4-minmax-rndnu-neondot.c",
+    "src/qu8-igemm/gen/2x16c4-minmax-fp32-neondot.c",
     "src/qu8-igemm/gen/2x16c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/2x32c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/3x8c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/3x16c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/3x32c4-minmax-rndnu-neondot.c",
     "src/qu8-igemm/gen/4x8c4-minmax-rndnu-neondot.c",
+    "src/qu8-igemm/gen/4x8c4-minmax-rndnu-neondot.c",
```

Comment:
Duplicate. Is it supposed to be "src/qu8-igemm/gen/4x8c4-minmax-fp32-neondot.c"?

Reply:
Good catch, yeah it's the latter. I never tested with Bazel, let me fix it, thanks!